backend/Dockerfile

```diff
@@ -1,7 +1,5 @@
-FROM node:16-alpine3.14
-# Installing libvips for sharp compatibility
-RUN apk --update-cache add vips-dev \
-  && rm -rf /var/cache/apk/*
+FROM node:18-alpine3.18
+RUN apk update && apk add python3 build-base
```

Comment:
I don't entirely understand why we removed 

```
# Installing libvips for sharp compatibility
RUN apk --update-cache add vips-dev \
  && rm -rf /var/cache/apk/*
```

and added

```
RUN apk update && apk add python3 build-base
```

I think we use docker to build Strapi, so I don't think these changes should have any runtime implications as long as the builds are still successful.

Reply:
Vips-dev was originally included because otherwise the yarn install command errored out (sharp dependency). 

Upon upgrading to Node 18, my intent was to declutter the install process by eliminating those additional packages, keeping the install process as barebone as possible.

However, running the build without any additional package produced errors since both Python and make commands were missing. That's why python and build-base were reintroduced.